astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
Right, I see that, but I'm saying it's a moot point anyway because most observers I know use "above airmass" and "below airmass" interchangeably, as it's usually clear from context what you really mean.  But as a *constraint* it makes more sense the way I was saying.